pyincore/analyses/montecarlolimitstateprobability/__init__.py

```diff
@@ -0,0 +1,9 @@
+# Copyright (c) 2019 University of Illinois and others. All rights reserved.
```

Comment:
minor nitpick - I think copyright should be 2024 maybe? I know the original was marked 2019, but maybe this is a dev discussion.

Reply:
Just leaving this here to close the loop. It was decided we should keep the original copyright.